third_party/caffe/Dockerfile

```diff
@@ -29,8 +29,9 @@ WORKDIR $CAFFE_ROOT
 
 # FIXME: use ARG instead of ENV once DockerHub supports this
 # https://github.com/docker/hub-feedback/issues/460
-# ENV CLONE_TAG=1.0
-ENV CLONE_TAG=master
+ENV CLONE_TAG=1.0
```

Comment:
@gasgallo It is better to squash the two commits to one, you can use the `git rebase` and `squash` command

Reply:
@gasgallo Sorry, I found that I can squash you commits by the merge operation on github, so there is no need for you to do that.